backend/src/main/scala/com/softwaremill/bootzooka/http/model/headers/headers.scala

```diff
@@ -0,0 +1,86 @@
+package com.softwaremill.bootzooka.http.model.headers
```

Comment:
Yea, about the packages, they are currently kind of a mess. There is this BCE pattern Adam Bien was trying to popularize and IMHO it's great, [you should look into it](http://www.adam-bien.com/roller/abien/entry/bureaucratic_design_with_java_ee).

Anyway, those HTTP headers are not an API, they are comming from non-functional requirement of security, so building upon the akka architecture and the BCE pattern the best place would be something like this: `boundary/http/headers/headers.scala`.
The main thing being here the fact that this code models HTTP not any business domain.